example/src/main/java/com/uploadcare/android/example/fragments/CdnFragment.kt

```diff
@@ -28,9 +29,24 @@ class CdnFragment : Fragment() {
 
     private val args: CdnFragmentArgs by navArgs()
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setHasOptionsMenu(true)
+    private val menuProvider = object : MenuProvider {
```

Comment:
Lets move it out to the separate file

Reply:
Or another way is implement `MenuProvider` by Fragment itself

Reply:
Realized via interface implementation by fragments, because it's easier to invoke viewModel and local methods without extra callbacks